src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PartATemplateReplacementServiceTest.kt

```diff
@@ -73,6 +74,10 @@ internal class PartATemplateReplacementServiceTest : ServiceTestBase() {
                 )
               )
             )
+          ),
+          underIntegratedOffenderManagement = UnderIntegratedOffenderManagement(
+            selected = "YES",
+            allOptions = listOf(TextValueOption(value = "YES", text = "Yes"), TextValueOption(value = "NO", text = "No"), TextValueOption(value = "NOT_APPLICABLE", text = "Not applicable"))
```

Comment:
Does the existing `hasVictimsInContactScheme` output 'N/A' or 'Not applicable' to the Part A doc? Prob best to stay consistent with that.

Reply:
Done :)